tpgtools/overrides/compute/beta/forwarding_rule.yaml

```diff
@@ -123,3 +123,6 @@
 - type: EXCLUDE
   field: location
   location: global
+- type: EXCLUDE
```

Comment:
This is why we prefer having split out resources for different locations, right?

Reply:
Yeah, absolutely.